crates/api_types/src/connector/snowflake_debezium.rs

```diff
@@ -0,0 +1,23 @@
+use crate::connector::ConstructPipelineConnector;
+use crate::service::ServiceConfig;
+use pipeline_types::config::ConnectorConfig;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+use utoipa::ToSchema;
+
+/// Configuration for a Snowflake-Debezium output connector.
```

Comment:
I don't think we use Debezium for Snowflake. We use Kafka Connect.

Reply:
I've for now removed this connector type to limit the scope of the PR.